provisioning/miniprov/hare_mp/store.py

```diff
@@ -44,6 +44,12 @@ def get_machine_id(self) -> str:
     def get_storage_set_index(self) -> int:
         raise NotImplementedError()
 
+    def get_machine_ids_for_service(self, service_type: str) -> List[str]:
```

Comment:
Lets please add some 2-3 liner documentation for all the new functions that are added in this PR.
Thanks!